CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ New Features:
 - Add baseUrl parameter to support slack-compatible integrations like Mattermost
   or Rocket Chat ([#293][#293])
 - Regression notification implemented ([#294][#294])
+- Add capability to include N lines from the tail of the console log ([#306][#306])
```

Comment:
Please do not modify the changelog.  For now, maintainers build the changelog when we're ready to do a release.  Also, your change is not in Slack Plugin 2.2 so it doesn't make sense to add it here.